microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/keycloak/KeycloakIntegrationHighLevelScenarioTest.java

```diff
@@ -51,7 +51,7 @@ public class KeycloakIntegrationHighLevelScenarioTest {
     private static final String KEYCLOAK_ADMIN_PASSWORD = "password";
 
     public static Docker keycloakContainer = new Docker.Builder(KEYCLOAK_CONTAINER_NAME,
-            "quay.io/keycloak/keycloak:latest")
+            "quay.io/keycloak/keycloak:24.0")
```

Comment:
with this we have come a full circle https://github.com/jboss-eap-qe/eap-microprofile-test-suite/pull/266

Reply:
Well, not actually full :)

Reply:
I agree that we could try and live with just latest though...

Reply:
yeah, but let's go with this quick fix while we plan to update later..